pyhf/modifiers/__init__.py

```diff
@@ -93,27 +116,28 @@ def modifier(*args, **kwargs):
     name = kwargs.pop('name', None)
     constrained = bool(kwargs.pop('constrained', False))
     shared = bool(kwargs.pop('shared', False))
+    pdf_type = str(kwargs.pop('pdf_type', 'normal'))
```

Comment:
we hardcode the default here as well as in `add_to_registry` , should we pick one or the other/

one way is to have 

`    pdf_type = str(kwargs.pop('pdf_type'))`
in the decorator and 

```
def ..(...pdf_type = None)
pdf_type = pdf_type or 'normal'
```

in the function

Reply:
`pdf_type = str(kwargs.pop('pdf_type'))` fails if `pdf_type` isn't specified.

Reply:
just realized we're doing the same with the kwargs above -- it's not a big issue so feel free to ignore.